src/api/test/functional/webui/webui_controller_test.rb

```diff
@@ -0,0 +1,42 @@
+require_relative '../../test_helper'
+
+class Webui::WebuiControllerTest < Webui::IntegrationTest
+
+  setup do
+    @before = CONFIG['proxy_auth_mode']
+    # Fake proxy mode
+    CONFIG['proxy_auth_mode'] = :on
+  end
+
+  teardown do
+    CONFIG['proxy_auth_mode'] = :off
```

Comment:
Oups. You are right.
And actually I could use stubs here. I'll rebase in a bit. Thanks!